src/test/plugin/fixtures/TrustSet.cpp

```diff
@@ -666,15 +931,21 @@ getTransactors()
          61,
          {formatPtr, 5},
          ConsequencesFactoryType::Normal,
-         NULL,
-         NULL,
+         nullptr,
+         nullptr,
          preflight,
          preclaim,
          doApply,
-         NULL,
-         NULL,
-         NULL,
-         NULL}};
+         nullptr,
+         nullptr,
+         nullptr,
+         nullptr,
+         payFee,
```

Comment:
Can these tests do more than just run the code? Ideally they would change something about these functions and make sure that that change is actually happening.

Reply:
done 81c8243ba